test/strings/types.jl

```diff
@@ -46,10 +55,12 @@ for idx in [0, 1, 4]
     @test SubString("∀∀", 4, idx) == "∀∀"[4:idx]
 end
 
-# second index beyond endof("∀∀")
-for idx in 5:8
+# index beyond endof("∀∀")
```

Comment:
"second" was accurate.

Reply:
I still don't get what it means.

Reply:
It refers to the second index (third argument) passed to `SubString`.

Reply:
Oh, that was completely unclear to me since the comment doesn't mention `SubString` at all. I think I'd like to deprecate the three-arg constructor for `SubString` anyway and always use `SubString(s, i:j)` instead.